app/models/order.rb

```diff
@@ -15,6 +15,10 @@ def total_quantity
     self.order_items.sum(:quantity)
   end
 
+  def self.sort_by_status
```

Comment:
Without a model test, our spec/models test coverage will drop for this but that's okay.

Reply:
Yeah, I thought it might redundant, especially given our time constraint. I tested the sorting with css on the feature test. Model coverage is at 98.41% .